beetsplug/deezer.py

```diff
@@ -75,14 +75,22 @@ def album_for_id(self, album_id):
         deezer_id = self._get_id("album", album_id, self.id_regex)
         if deezer_id is None:
             return None
-
-        album_data = requests.get(self.album_url + deezer_id).json()
-        if "error" in album_data:
-            self._log.debug(
-                f"Error fetching album {album_id}: "
-                f"{album_data['error']['message']}"
-            )
+        try:
+            album_data = requests.get(
+                self.album_url + deezer_id, timeout=10
+            ).json()
+        except (
+            requests.exceptions.ConnectionError,
+            requests.exceptions.RequestException,
+        ) as e:
```

Comment:
This could be made shorter as:

```
except requests.ConnectionError as e:
```

That is, `ConnectionError` is already a subclass of `RequestException`, so that suffices. And the class is exported as `requests.ConnectionError` (no need to use the `exceptions` module explicitly).